core/src/main/java/net/adoptopenjdk/icedteaweb/client/console/JavaConsole.java

```diff
@@ -452,47 +440,20 @@ public void run() {
         });
     }
 
-    public void hideConsoleLater() {
-        SwingUtils.invokeLater(new Runnable() {
-
-            @Override
-            public void run() {
-                JavaConsole.getConsole().hideConsole();
-            }
-        });
-    }
-
-    protected void printSystemProperties() {
+    private void printSystemProperties() {
 
         LOG.info(" ----");
         LOG.info("System Properties:");
         LOG.info("");
-        Properties p = System.getProperties();
-        Set<Object> keys = p.keySet();
-        for (Object key : keys) {
+        final Properties p = System.getProperties();
+        for (Object key : p.keySet()) {
             LOG.info(key.toString() + ": " + p.get(key));
         }
         LOG.info(" ----");
     }
 
-    public void setClassLoaderInfoProvider(ClassLoaderInfoProvider clip) {
-        classLoaderInfoProvider = clip;
-    }
-
     private void printClassLoaders() {
-        if (classLoaderInfoProvider == null) {
-            LOG.debug("No Classloader info exists in system");
-        } else {
-            LOG.debug(" ----");
-            LOG.debug("Available Classloaders: ");
-            Set<String> loaders = classLoaderInfoProvider.getLoaderInfo().keySet();
-            for (String loader : loaders) {
-                LOG.debug(loader + "\n"
-                        + "  codebase = "
-                        + classLoaderInfoProvider.getLoaderInfo().get(loader));
-            }
-            LOG.debug(" ----");
-        }
+        LOG.debug("No Classloader info exists in system");
```

Comment:
Ugh. Why so?  Class laoders info are part of proprieatary javaws. It should be fixed rather then removed.

Reply:
Fixing is non trivial as there is currently no implementation of the interface `ClassLoaderInfoProvider`. I can open an issue about the missing implementation.